### ISSUE TITLE
About the accuracy on cityScapes to foggyCityScapes

### ISSUE THREAD

**Issue by lch1999** [kinredon/umt#17]
I train and test the model on cityScapes to foggyCityScapes，but my mAP is 35.1, which is lower than 41.7 reported in the paper. I guess maybe there are some mistakes in my configs.

Here are my configs:
Called with args:
Namespace(aug=True, batch_size=1, binary=False, budget=0.3, checkepoch=1, checkpoint=0, checkpoint_interval=10000, checksession=1, class_agnostic=False, conf=True, conf_gamma=0.1, cuda=True, dataset='cityscape', dataset_t='foggy_cityscape', dc=None, detach=True, disp_interval=100, ef=False, eta=0.1, gamma=5, gc=False, image_dir='images', lam=0.01, lam2=0.1, large_scale=False, lc=False, load_name='models', load_name_conf=None, lr=0.001, lr_decay_gamma=0.1, lr_decay_step=5, mGPUs=False, max_epochs=8, net='vgg16', num_workers=0, optimizer='sgd', pl=True, pretrained_epoch=0, resume=False, save_dir='models', session=1, source_like=True, source_model=None, start_epoch=1, student_load_name='models', target_like=True, teacher_alpha=0.99, teacher_load_name='models', test_results_dir='test_results', threshold=0.8, use_tfboard=False, vis=False)
Using config:
{'ANCHOR_RATIOS': [0.5, 1, 2],
 'ANCHOR_SCALES': [8, 16, 32],
 'CROP_RESIZE_WITH_MAX_POOL': False,
 'CUDA': False,
 'DATA_DIR': '/home/lch1999/myModel/data',
 'DEDUP_BOXES': 0.0625,
 'EPS': 1e-14,
 'EXP_DIR': 'vgg16',
 'FEAT_STRIDE': [16],
 'GPU_ID': 0,
 'MATLAB': 'matlab',
 'MAX_NUM_GT_BOXES': 30,
 'MOBILENET': {'DEPTH_MULTIPLIER': 1.0,
               'FIXED_LAYERS': 5,
               'REGU_DEPTH': False,
               'WEIGHT_DECAY': 4e-05},
 'PIXEL_MEANS': array([[[102.9801, 115.9465, 122.7717]]]),
 'POOLING_MODE': 'align',
 'POOLING_SIZE': 7,
 'RESNET': {'FIXED_BLOCKS': 1, 'MAX_POOL': False},
 'RESNET101_PATH': 'data/pretrained_model/resnet101_caffe.pth',
 'RESNET50_PATH': 'data/pretrained_model/resnet50_caffe.pth',
 'RNG_SEED': 3,
 'ROOT_DIR': '/home/lch1999/myModel',
 'TEST': {'BBOX_REG': True,
          'HAS_RPN': True,
          'MAX_SIZE': 1200,
          'MODE': 'nms',
          'NMS': 0.3,
          'PROPOSAL_METHOD': 'gt',
          'RPN_MIN_SIZE': 16,
          'RPN_NMS_THRESH': 0.7,
          'RPN_POST_NMS_TOP_N': 300,
          'RPN_PRE_NMS_TOP_N': 6000,
          'RPN_TOP_N': 5000,
          'SCALES': [600],
          'SVM': False},
 'TRAIN': {'ASPECT_GROUPING': False,
           'BATCH_SIZE': 256,
           'BBOX_INSIDE_WEIGHTS': [1.0, 1.0, 1.0, 1.0],
           'BBOX_NORMALIZE_MEANS': [0.0, 0.0, 0.0, 0.0],
           'BBOX_NORMALIZE_STDS': [0.1, 0.1, 0.2, 0.2],
           'BBOX_NORMALIZE_TARGETS': True,
           'BBOX_NORMALIZE_TARGETS_PRECOMPUTED': True,
           'BBOX_REG': True,
           'BBOX_THRESH': 0.5,
           'BG_THRESH_HI': 0.5,
           'BG_THRESH_LO': 0.0,
           'BIAS_DECAY': False,
           'BN_TRAIN': False,
           'DISPLAY': 10,
           'DOUBLE_BIAS': True,
           'FG_FRACTION': 0.25,
           'FG_THRESH': 0.5,
           'GAMMA': 0.1,
           'HAS_RPN': True,
           'IMS_PER_BATCH': 1,
           'LEARNING_RATE': 0.001,
           'MAX_SIZE': 1200,
           'MOMENTUM': 0.9,
           'PROPOSAL_METHOD': 'gt',
           'RPN_BATCHSIZE': 256,
           'RPN_BBOX_INSIDE_WEIGHTS': [1.0, 1.0, 1.0, 1.0],
           'RPN_CLOBBER_POSITIVES': False,
           'RPN_FG_FRACTION': 0.5,
           'RPN_MIN_SIZE': 8,
           'RPN_NEGATIVE_OVERLAP': 0.3,
           'RPN_NMS_THRESH': 0.7,
           'RPN_POSITIVE_OVERLAP': 0.7,
           'RPN_POSITIVE_WEIGHT': -1.0,
           'RPN_POST_NMS_TOP_N': 2000,
           'RPN_POST_NMS_TOP_N_TARGET': 256,
           'RPN_PRE_NMS_TOP_N': 12000,
           'SCALES': [600],
           'SNAPSHOT_ITERS': 5000,
           'SNAPSHOT_KEPT': 3,
           'SNAPSHOT_PREFIX': 'res101_faster_rcnn',
           'STEPSIZE': [30000],
           'SUMMARY_INTERVAL': 180,
           'TRIM_HEIGHT': 600,
           'TRIM_WIDTH': 600,
           'TRUNCATED': False,
           'USE_ALL_GT': True,
           'USE_FLIPPED': True,
           'USE_GT': False,
           'WEIGHT_DECAY': 0.0005},
 'USE_GPU_NMS': True,
 'VGG_PATH': '/home/lch1999/pretrainvggforUMT/vgg16_caffe.pth'}

And here are my results:
AP for bus = 0.3952
AP for bicycle = 0.3204
AP for car = 0.5049
AP for motorcycle = 0.2490
AP for person = 0.3292
AP for rider = 0.4064
AP for train = 0.3154
AP for truck = 0.2855
Mean AP = 0.3508

Could you please check my config and give some suggestions？Thank you very much!
